tests/models/test_ease.py

```diff
@@ -27,6 +27,31 @@
 
 
 class TestEASEModel:
+    def test_from_config(self) -> None:
```

Comment:
Please make the tests in the same way as they are done for knn and also models:  
1. Separate class
2. Add `test_get_config_and_from_config_compatibility` and `test_default_config_and_default_model_params_are_the_same`. For these 2 tests I think it makes sense to add some common functions to utils, like `assert_second_fit_refits_model`

Reply:
Separate class was created for each model as per provided example. 
2 functions were added to `tests/utils.py` to cover compatibility and similarity tests.

Reply:
Great! Could you please use your new functions in ALS and KNN model tests?